chore: added updateThread, getSectionById, createNewTopic

Gukkey opened the pull request:
- thread's updatedAt column is updated whenever a reply is added

- added method in action.tsx to create new topic

- added method for updating threads

Gukkey committed:
chore: added updateThreadTimeStamp, getSectionById, createNewTopic

- thread's updatedAt column is updated whenever a reply is added

- added method in action.tsx to create new topic

shiyaamsunder commented on src/db/queries.ts:
```diff
@@ -21,6 +25,17 @@ export async function createDiscussionThread(data: Prisma.ThreadUncheckedCreateI
   return await prisma.thread.create({ data })
 }
 
+export async function updateThreadTimeStamp(data: Prisma.ThreadUncheckedUpdateInput) {
```
change the function name to `updateThread` and inside the function use the `data` directly

Gukkey replied:
done

Gukkey committed:
chore: updateThreadTimestamp() -> updateThread

- changed updatedThreadTimestamp to updatedThread to accommodate all kinds of thread changes
